backend/scaife_stack_atlas/extractors/extract_lexicon_thucydideum.py

```diff
@@ -0,0 +1,137 @@
+import itertools
+import json
+from functools import lru_cache
+from pathlib import Path
+
+import jsonlines
+from lxml import etree
+from more_itertools import peekable
+
+
+CATALOG_API_HOST = "catalog-api-dev.scaife.eldarion.com"
+
+DICTIONARY_PATH = Path("data/annotations/dictionaries/lexicon-thucydideum")
```

Comment:
Longer term, I would like this to be loaded from https://github.com/gregorycrane/Thucydides-new-working-materials directly.